### PR TITLE
JS modules

### PR DESCRIPTION
**Pull request by zkr024**
split the original js code using modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+- Luxon
+- Check the code with Linters tests:
+  _Webhint_
+  _Stylelint_
+  _Lighthouse_
+  _esLint_
+
+## Live Demo
+
+[Live Demo](https://zkr024.github.io/modules/)
```
Comment (Deepakdanger) [inline README.md]: - [ ] Kindly update your live link.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @zkr024, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] After adding the `book` to the `List` page, The `Remove` button is not working.Kindly fix it.
- **Required** the app is working - it is possible to add and remove books from the list.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] There should be a `.gitignore` file to ignore `node modules`.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (zkr024) [reply README.md]: thank you @Deepakdanger for the feedback.
I'll work on it...but I have a question. It works from the vs code go live option, but if I try to open the index.html file, it prompts this error:  
'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome, chrome-extension, chrome-untrusted, https. 
Any suggestion there maybe

**Review by AnselemOdims — APPROVED** [review]
### Hi  @zkr024 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**